README.md

```diff
@@ -151,21 +144,113 @@ domain_beanPropertyValue1_key1_key2_...keyN_attrName{beanpropertyName2="beanProp
 ```
 If a given part isn't set, it'll be excluded.
 
-## Integration Testing
+## HTTP Authentication (optional)
+
+HTTP BASIC authentication it supported using various configuration formats.
+
+---
+
+Simple example (plaintext configuration value):
```

Comment:
I think rather than calling the examples _simple_ and _more secure_ we should call them something like
* Store the plaintext password in the configuration file
* Store the SHA-256 hash of the password in the configuration file
* Store the PBKDF hash of the password in the configuration file

Reply:
Reworded README.md